monkey/monkey_island/cc/services/attack/technique_reports/pba_technique.py

```diff
@@ -0,0 +1,54 @@
+import abc
+
+from common.utils.attack_utils import ScanStatus
+from monkey_island.cc.database import mongo
+from monkey_island.cc.services.attack.attack_config import AttackConfig
+from monkey_island.cc.services.attack.technique_reports import AttackTechnique
+
+
+class PostBreachTechnique(AttackTechnique, metaclass=abc.ABCMeta):
+    """ Class for ATT&CK report components of post-breach actions """
+
+    @property
+    @abc.abstractmethod
+    def pba_names(self):
+        """
+        :return: name of post breach action
```

Comment:
So `name` or `names`? Does the user expect array or a string?

Reply:
As I later found it's array of strings. Can we add `-> List[str]`?

Reply:
Done